src/Domains/Inventory/Variants/Actions/UpdateToWarehouseAction.php

```diff
@@ -60,6 +62,13 @@ public function execute(): Variants
             ))->execute();
         }
 
+        if ($this->variantsWarehousesDto->price && $oldPrice !== $this->variantsWarehouses->price) {
```

Comment:
@arfenis we can move this to a observer , so its tied to the variant warehouse logic? in aftersave or something like taht